pmlc/conversion/tile_to_pxa/tile_to_pxa.cc

```diff
@@ -586,13 +580,13 @@ struct ContractionOpConversion : public OpConversionPattern<ContractionOp> {
     // TODO: addInitializer
 
     // Make the outer loops
-    auto dynamicRanges = ArrayRef<Value>();
-    auto forOp = rewriter.create<pxa::AffineParallelForOp>(loc, rewriter.getI64ArrayAttr(ranges), dynamicRanges);
-    auto body = rewriter.createBlock(&forOp.inner());
+    auto forOp = rewriter.create<pxa::AffineParallelForOp>(loc, ranges);
+    auto body = forOp.getBody();
+    rewriter.setInsertionPointToStart(body);
```

Comment:
Can we use `rewriter.createBlock`? I think it internally sets the insertion point.

Reply:
The affine for already has a block, and createBlock was incorrectly adding another one.